indico/modules/events/management/templates/_management_frame.html

```diff
@@ -24,7 +24,7 @@
 {%- endmacro -%}
 
 
-{% macro render_event_management_header_right(event, event_types) %}
+{% macro render_event_management_header_right(event, event_types, with_clone_info=true, with_action_menu=true) %}
```

Comment:
Maybe add a comment right above it so the args aren't accidentally removed at some point?

```suggestion
{# The extra args are not used in Indico but are there on purpose for others overriding the template. #}
{% macro render_event_management_header_right(event, event_types, with_clone_info=true, with_action_menu=true) %}
```